services/install-minetest-server.sh

```diff
@@ -0,0 +1,261 @@
+#!/bin/bash
+
+function install {
+  # create service directory
+  mkdir -p /srv/minetest-server
+
+  # create yml(s)
+  {
+    echo "version: \"2.1\""
+    echo "services:"
+    echo "  minetest:"
+    echo "    image: linuxserver/minetest:latest"
+    echo "    environment:"
+    echo "      - PUID=1000"
+    echo "      - PGID=1000"
+    echo "      - TZ=Europe/London"
+    echo "      - CLI_ARGS=\" --gameid minetest --worldname world\""
```

Comment:
Need to replace these environment variables with `- PUID=\${PUID}` or whatever you want to call the variable in the .env file so that it reads the values from the .env file.

Reply:
Okay. I changed it.